src/i_system.c

```diff
@@ -44,6 +44,84 @@ ticcmd_t *I_BaseTiccmd(void)
 // I_Error
 //
 
+#ifdef WOOF_DEBUG
+
+#if defined(_WIN32)
+#include <dbghelp.h>
+
+static void PrintBacktrace(void)
+{
+    HANDLE process = GetCurrentProcess();
+
+    boolean symbol_available = false;
+
+    if (SymInitialize(process, NULL, TRUE))
+    {
+        SymSetOptions(SYMOPT_UNDNAME);
+        symbol_available = true;
+    }
+
+    // Capture stack frames
+    void *stack[100];
+    USHORT frames = CaptureStackBackTrace(2, 100, stack, NULL);
+
+    // Print backtrace
+    I_Printf(VB_INFO, "Backtrace (frames %d):", frames);
```

Comment:
`(%d frames)`?

Reply:
This is the number of stack frames. We don't need this?

Reply:
No, it's fine. I would just reverse the number and the string.